winlogbeat/sys/wineventlog/renderer.go

```diff
@@ -222,6 +222,7 @@ func (r *Renderer) renderSystem(handle EvtHandle, event *winevent.Event) error {
 			event.Channel = data.(string)
 		case EvtSystemComputer:
 			event.Computer = data.(string)
+			// TODO: set event.Computer to FQDN if features.FQDN() is true?
```

Comment:
@efd6 @andrewkroh In this PR we're trying to set `host.name` in events generated by Beats to the FQDN if the Beat config contains `features.fqdn.enabled: true`.  What's the best way to retrieve the FQDN on a Windows host? I assume this is the line in code where we'd want to set `event.Computer` to the FQDN if the feature is enabled?

Reply:
Thanks for the background and suggestions, @andrewkroh.  I've created a separate issue to handle FQDN reporting for Windows hosts, as the use case urgently driving the FQDN feature implementation is for Beats running in containerized environments, which tend to be *nix most of the times.  I've created https://github.com/elastic/beats/issues/34782 to follow up on implementing FQDN reporting for Windows hosts.

Reply:
Do we need to keep this TODO in the code? From our previous discussion I believe we concluded we likely don't need to touch this.

Reply:
Good catch, we don't need this TODO any more.  Removing...